staging/src/k8s.io/component-base/cli/flag/flags.go

```diff
@@ -54,13 +54,13 @@ func InitFlags() {
 	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
 	pflag.Parse()
 	pflag.VisitAll(func(flag *pflag.Flag) {
-		klog.V(2).Infof("FLAG: --%s=%q", flag.Name, flag.Value)
+		klog.V(2).InfoS("FLAG", "flagName", flag.Name, "flagValue", flag.Value)
```

Comment:
It would be good to dump an example of what the new output looks like compared to the old one.